test/unit/app/browser/reducers/sitesReducerTest.js

```diff
@@ -149,8 +149,8 @@ describe('sitesReducerTest', function () {
       moveAction.prepend = true
       newState = sitesReducer(Immutable.fromJS(newState), moveAction).toJS()
       assert.equal(Object.keys(newState.sites).length, 3)
-      assert.equal(Object.values(newState.sites)[2].location, url)
-      assert.equal(Object.values(newState.sites)[2].order, 1)
+      assert.equal(Object.values(newState.sites)[1].location, url)
```

Comment:
If this is correct then I would add some description about it beside this comment, otherwise it could be confusing